src/openapi.yaml

```diff
@@ -2863,6 +2863,112 @@ paths:
               $ref: '#/components/headers/X-Is-Form-Designer'
             Content-Language:
               $ref: '#/components/headers/Content-Language'
+  /api/v2/logic/description:
+    post:
+      operationId: logic_description_create
+      description: |-
+        Generate a more human-readable description of a given JsonLogic expression.
+
+        This endpoint takes into account (optionally) client-provided sub-expression
+        descriptions via the ``_meta.description`` path. E.g. an input like:
+
+        ```json
+        {
+          "==": [
+            {"var": "foo"},
+            {
+                "+": [{"var": "a", "var": "b"}],
```

Comment:
Shouldn't this be `[{"var": "a"}, {"var": "b"}]`?

Reply:
yes, nice catch!